datamonster_api/lib/datamonster.py

```diff
@@ -211,35 +211,33 @@ def get_data(
         # todo: support multiple companies
         self._check_param(company=company, datasource=datasource)
 
-        params = {"companyId": company.id}
+        params = {"section_pk": company.id}
 
         if start_date is not None:
-            params["startDate"] = start_date
+            if not datasource.upperDateField:
+                raise DataMonsterError(
+                    "This data source does not yet support date queries"
```

Comment:
I'd take out the 'yet' here and in the `end_date` check.

Reply:
updated